src/tfr_description/xacro/proto_drivebase_controller.xacro

```diff
@@ -0,0 +1,49 @@
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="proto">
+  <!--xacro dectiptio of the prototype skid steer controller-->
```

Comment:
Typo: I'm not sure, but is this supposed to be "depiction"?

Reply:
depiction, I'll fix it